docs/building.md

```diff
@@ -4,7 +4,7 @@ category: 6255e67693d5200013b1fa41
 hidden: false
 ---
 
-Phylum is written in Rust, so you'll need a recent Rust installation to build it (we tested with v1.58.0). [Install Rust](https://www.rust-lang.org/tools/install)
+Phylum is written in Rust, so you'll need a recent Rust installation to build it (we tested with v1.61.0). [Install Rust](https://www.rust-lang.org/tools/install)
```

Comment:
v1.61.0 is slightly newer than the v1.60 specified in `cli/cargo.toml`. The guess is that this is intentional (testing was done on a newer version than the minimum one *required*), but pointing it out anyway.

Reply:
Yep. 1.61 is what I've got installed, so it's what I tested with.